response_operations_ui/views/info.py

```diff
@@ -1,18 +1,26 @@
-import json
+from json import JSONDecodeError, loads
+import logging
 from pathlib import Path
 
 from flask import make_response, jsonify
+from structlog import wrap_logger
 
 from response_operations_ui import app
 
 
+logger = wrap_logger(logging.getLogger(__name__))
+
+
 @app.route('/info', methods=['GET'])
 def get_info():
 
     _health_check = {}
     if Path('git_info').exists():
         with open('git_info') as io:
-            _health_check = json.loads(io.read())
+            try:
+                _health_check = loads(io.read())
+            except JSONDecodeError as e:
+                logger.exception('Failed to decode git_info json', exception=e)
```

Comment:
You don't need the exception kwarg as the exception level logger will print any `exc_info` present anyway. See https://github.com/ONSdigital/eq-survey-runner/blob/e920e1dbfc633e7e6f6ed25002f139672674a8ae/application.py#L47 for an example of how to get the exception into the logged event in a splunk friendly way using `format_exc_info`.